mitodl/open-discussions
# Fix OLL logo image

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes https://trello.com/c/YkFGXP6a/237-bug-wrong-logo-for-oll-course

#### What's this PR do?
Replaces the MIT Open image being used as the OLL logo with a correct version, resized to match the old image's size.

#### How should this be manually tested?
Go to `/learn/search` and filter on "Open Learning Library" under the "Offered By" facet. You should see an image similar to the one in the screenshot.

#### Screenshots
![Screenshot from 2020-03-27 15-49-53](https://user-images.githubusercontent.com/863262/77794659-a2d14e80-7042-11ea-939c-cd76c5dcf556.png)
